apps/storybook/src/Toolbar.stories.tsx

```diff
@@ -76,7 +76,13 @@ export const Default = {
           />
 
           <Separator />
-          <GridToggler value={showGrid} onToggle={toggleGrid} />
```

Comment:
A bunch of other toggles don't have their own components, so it didn't seem to make sense to still have `GridToggler` and `FlipYAxisToggler`. I decided to deprecate them rather than to create more such components (like `FlipXAxisToggler`) or to turn `FlipYAxisToggler` into a more generic `FlipAxisToggler`.